src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -1,150 +1,155 @@
 <template>
   <div class="tree-builder-toolbar">
```

Comment:
The diff seems hard to follow here, but what I did in this template was very simple:
- renamed the class `tree-builder-toolbar` into `tree-builder-toolbar-main-menu`
- created a new div wrapper, and named it `tree-builder-toolbar`
- put the new alert banner within this new added div `tree-builder-toolbar` (check here: https://github.com/FLOIP/flow-builder/pull/92/files#diff-3db5c56652da6d4f7b33c23e432a38b9b73d0b30d47df648929798e4c82d498bR147)